register_test.go

```diff
@@ -0,0 +1,51 @@
+package currency
```

Comment:
This library uses black box testing, all other tests are in `currency_test`, so this one needs to be as well.